tests/hpc/mrsh_master.pm

```diff
@@ -28,15 +28,9 @@ sub run {
     barrier_create("SLAVE_MRLOGIN_STARTED",      2);
     barrier_create("MRSH_MASTER_DONE",           2);
 
-    select_console 'root-console';
```

Comment:
why you remove it here , but leave it for mrsh_slave ?

Reply:
I assume copy-paste error - fixing